### PR TITLE
Script to load predictive rates from csv

### PR DESCRIPTION
**Pull request by alanren2**
Needs to have the csv in the same directory. Not intended for legacy use.

### CODE REVIEW

File: backend/scripts/upload_csv.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        idx_idx = 0
+        id_idx = 1
+        treat_idx = 3
+        mean_rate_idx = 4
+        sd_idx = 5
+        county_idx = 7
+        num_days_idx = 12
+        start_days = 10
+        count = 0
+        print(len(rows))
```
Comment (caiyingshen) [inline backend/scripts/upload_csv.py]: :(

File: backend/scripts/upload_csv.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        responses = {}
+        for row in rows:
+            count += 1
+            if count % 10000 == 0:
+                print(count)
```
Comment (caiyingshen) [inline backend/scripts/upload_csv.py]: :(

File: backend/scripts/upload_csv.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    rates={"2020": {str(days_after): [float(row[mean_rate_idx]), float(row[sd_idx]), days_after]}},
+                )
+            if r.tract_id in responses:
+                existing = responses[r.tract_id]
+                r.update(existing)
+                responses[r.tract_id] = r
+            else:
+                responses[r.tract_id] = r
+        
+        print(len(responses.values()))
```
Comment (caiyingshen) [inline backend/scripts/upload_csv.py]: :(

File: backend/scripts/upload_csv.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        count = 0
+        for r in responses.values():
+            count += 1
+            if count % 100 == 0:
+                print(count)
```
Comment (caiyingshen) [inline backend/scripts/upload_csv.py]: :(

**Review by caiyingshen — CHANGES_REQUESTED** [review]

**Review by caiyingshen — APPROVED** [review]
beep BEEP

**Review by chyku — APPROVED** [review]

**Pull request merged**